cmd/ccboc/cmd/commands.go

```diff
@@ -364,6 +366,90 @@ func request(method, endpoint string, buffer *bytes.Buffer) ([]byte, *errorRespo
 	return body, nil, nil
 }
 
+func getCalculationPhase() error {
+	for {
```

Comment:
This is going to spam the API. Can we have a refresh rate here instead of an infinite loop?

Reply:
It doesnt actually spam the API. We only refresh whenever we hit the ENTER key.

Reply:
This is an attack factor. Instead of hitting the ENTER key we should allow the user to specify an interval, and have as default 3 seconds maybe?

Reply:
Good point, didnt think of that! Is adding a sleep for 3s sufficient?
The issue is, that the `Run()` method is blocking the code until `Stop()` comes in.

`
if key == tcell.KeyEnter { // to refresh the table contents
	app.Stop()
	time.Sleep(time.Second * 3)
}
`